### DISCUSSION TITLE
Add Possibility to Swap Command and Option Keystrokes Sent to Remote System

### DISCUSSION THREAD

**Discussion by disbolog** [rustdesk/rustdesk#4439] [Feature Request]
I am using RustDesk on Mac, with remote system being an i3wm-enabled Linux.

 `Command`+`Shift`+`Q` is a system shortcut for Logout on Mac.

It also maps to `Super`+`Shift`+`Q` which is often used for "Exit Window" shortcut on WMs such as i3.

Please, add an option to swap `Command` with `Option` for the remote system, so it would be possible to map `Option`+`Shift`+`Q` to `Super`+`Shift`+`Q`, which eliminates 2 issues at the same time:
1. No more unwanted prompt to log off on Mac each time you press such a combination;
2. `Super` now finds itself in a traditional spot for Linux/PC keyboards - inbetween `Control` and `Alt`.

Thank you in advance.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/3121

**Reply by disbolog** [reply]
Thank you, but this, however, swaps `Control` and `Command`, not `Option` and `Command`, so the feature request stands valid.

**Reply by rustdesk** [reply]
Yes, but we have no plan yet. 
https://github.com/rustdesk/rustdesk/discussions/4420 is a good example for such simple feature request.

**Reply by LuSrackhall** [reply]
+1
#14696

#14681

**Comment by zacaj** [comment]
See also https://github.com/rustdesk/rustdesk/discussions/11466
